doc/nrf/libraries/bin/lwm2m_carrier/app_integration.rst

```diff
@@ -130,6 +130,11 @@ Following are some of the general Kconfig options that you can configure:
   * This configuration is only supported for Push delivery method of firmware images.
   * Leaving this configuration empty (``0``) disables the timer for unknown subscriber IDs, and set it to 30 minutes for the SoftBank subscriber ID.
 
+* :kconfig:option:`CONFIG_LWM2M_CARRIER_QUEUE_MODE`
+
+  * This configuration specifies whether the LwM2M device is to inform the LwM2M server that it may be disconnected for an extended period of time.
```

Comment:
```suggestion
  * This configuration specifies whether the LwM2M device is to inform the LwM2M Server that it may be disconnected for an extended period of time.
```
As per the specs.